yara will fail pkgconf test pipeline

If the pkgconf test pipeline is added to yara it will fail in the following way:

```
2024/11/12 23:16:15 INFO running step "test/pkgconf"
2024/11/12 23:16:15 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:16:15 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:16:15 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:16:15 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:16:15 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + basename /home/build/melange-out/yara-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + dev_pkg=yara-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + apk info -L yara-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + grep -q ^Name: usr/lib/pkgconfig/yara.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + basename usr/lib/pkgconfig/yara.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + lib_name=yara uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + echo usr/lib/pkgconfig/yara.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + grep -q '^usr/lib/pkgconfig/yara.pc$\|^usr/share/pkgconfig/yara.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + pkgconf --exists yara uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + grep -q ^Version: usr/lib/pkgconfig/yara.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + pkgconf --modversion yara uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 INFO 4.5.2 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + grep -q ^Libs: usr/lib/pkgconfig/yara.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + pkgconf --libs yara uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 INFO -lyara uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + grep -q ^Cflags: usr/lib/pkgconfig/yara.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN + pkgconf --cflags yara uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN Package libcrypto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN Perhaps you should add the directory containing `libcrypto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 WARN Package 'libcrypto', required by 'yara', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:15 INFO pod ad950569c4f90f5772d4d61282199e006056c2a334c26a1070ce4f6cf0a21c3c terminated
2024/11/12 23:16:15 INFO pod 807a7ad8445cc54f5b4f5c69ab00a0524a6ad4f7c5782f6ac999c784657878eb terminated
2024/11/12 23:16:15 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:16:15 INFO   workspace dir: /home/user/tmp/melange-workspace-1650004015
2024/11/12 23:16:15 INFO   guest dir: /home/user/tmp/melange-guest-4054784640
2024/11/12 23:16:15 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/yara] Error 1
```

--- a/yara.yaml
+++ b/yara.yaml
@@ -2,7 +2,7 @@
 package:
   name: yara
   version: 4.5.2
-  epoch: 1
+  epoch: 2
   description: The pattern matching swiss knife for malware researchers
   copyright:
     - license: BSD-3-Clause
@@ -54,6 +54,9 @@ subpackages:
     dependencies:
       runtime:
         - yara
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: yara-doc
     description: yara manpages

